pkg/scd/server.go

```diff
@@ -69,3 +71,18 @@ func (a *Server) AuthScopes() map[auth.Operation]auth.KeyClaimedScopesValidator
 func (a *Server) MakeDssReport(ctx context.Context, req *scdpb.MakeDssReportRequest) (*scdpb.ErrorReport, error) {
 	return nil, stacktrace.NewErrorWithCode(dsserr.BadRequest, "Not yet implemented")
 }
+
+func incrementNotificationIndices(ctx context.Context, r repos.Repository, subs []*scdmodels.Subscription) error {
```

Comment:
nit: It seems like we want to introduce
```
package models

type Subscriptions []*Subscription

func (subs Subscriptions) IncrementNotificationIndices(context.Context, repos.Repository) error {
  ...
}
```

Reply:
Nice :) Done.

Reply:
Actually, only sort of nice.  Can't define that function in models because repos depends on models and that would create a circular dependency.  So, next best is to put it as a utility function in repos rather than models.  But that also requires the Subscriptions type to be defined there, and that seems like a non-ideal place to define it.  And then, we have to change all the explicit []*scdmodels.Subscription type declarations to the more opaque repos.Subscriptions type.  I'm not sure this actually increases overall clarity.

Reply:
Ah, the circular dependency issue is a deal breaker, indeed.